graspy/match/gmp.py

```diff
@@ -79,6 +79,9 @@ class GraphMatch(BaseEstimator):
     score_ : float
         The objective function value of for the optimal permutation found.
 
+    n_iter_ : int
+        Number of iterations run.
```

Comment:
```suggestion
        Number of Frank-Wolfe iterations run.
```
Add the following sentence as well: "If `n_init` > 1, `n_iter_` refelects the number of iterations performed at the initialization returned."

Reply:
updated!